README.md

```diff
@@ -9,9 +9,17 @@
 **Database Setup**
```

Comment:
there should be instructions here about setting up the venv

Reply:
There are. They're at the beginning of the README with a link to instructions on how to install python3 and pip3 with pyenv and setting those as the global versions so you don't have to type `python3` and `pip3` on every commands, or active or deactivate a virtual environment.